pyproject.toml

```diff
@@ -39,6 +39,7 @@ dependencies = [
   # direct dependencies
   "build",
   "click >= 8",
+  "importlib-metadata == 1.4; python_version < '3.8'",
```

Comment:
Why this specific version? Normally, you shouldn't restrict library runtime deps.

Reply:
This was the version in Python 3.8, the first time it was included as stdlib. Because it is being used with Python 3.7, I thought we should keep it as close as possible. Let me know if you want me to remove the limit.

Reply:
I removed the dependency on importlib now. We just don't get type hints in a very tiny part of the code, but that's OK.